### PR TITLE
OCPBUGS-43884:Incorrect YAML indentation

### PR DESCRIPTION
**Pull request by skrthomas**
Version(s):

4.12+
Issue:

https://issues.redhat.com/browse/OCPBUGS-43884
Link to docs preview:

https://84177--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/network_observability/network-observability-secondary-networks.html
QE review:
- [X] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by memodi — APPROVED** [review]
/label qe-approved

**Review by cbippley — COMMENTED** [review]
LGTM!

File: modules/network-observability-virtualization-configuration.adoc [hunk 1]
```diff
@@ -65,9 +65,9 @@ kind: FlowCollector
 metadata:
   name: cluster
 spec:
-# ...
-ebpf:
-    privileged: true              <1>
+  agent:
+    ebpf:
+      privileged: true              <1>
```
Comment (stleerh) [inline modules/network-observability-virtualization-configuration.adoc:70]: Move the "\<1>" back two spaces to align with the other numbers.

**Review by stleerh — COMMENTED** [review]
/lgtm

**Pull request merged**